.github/mergify.yml

```diff
@@ -0,0 +1,16 @@
+---
+# see https://docs.mergify.com/configuration/
+pull_request_rules:
+  - name: "Update out-of-date ready PRs"
+    conditions:
```

Comment:
There is `-or` operator in the docs (https://docs.mergify.com/conditions/#combining-conditions-with-operators), I'd suggest using it to be explicit.

Reply:
Or in our case, it's rather `and`...